source/accessibility/index.html.md.erb

```diff
@@ -47,21 +49,30 @@ The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
 
 ##### Compliance status
 
-This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 2 September 2020. It was last reviewed on 29 March 2021.
+This statement was prepared on 2 September 2020. It was last reviewed on 18 June 2021.
 
 This website was last tested in March 2021.
 
 ## Using the Technical Documentation Template for your own documentation
 
-This [Technical Documentation Template](https://github.com/alphagov/tech-docs-template) was last tested in March 2021.
+This [Technical Documentation Template](https://github.com/alphagov/tech-docs-template) was last tested in April 2021. The test was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
+
+DAC tested the [GovWifi technical documentation](https://docs.wifi.service.gov.uk/), which uses this Technical Documentation Template.
+
+This Technical Documentation Template is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.
+
+- In some circumstances, the main page heading is automatically focused when the page loads. This means keyboard-only users may miss the navigation elements. This fails [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).
```

Comment:
Would it be more natural to write 'in focus' rather than 'focused' or is this the terminology that is normally used?

Reply:
agree